package.json

```diff
@@ -89,7 +89,8 @@
     "stylelint-config-terra": "^2.0.0",
     "terra-dev-site": "^4.0.0",
     "terra-enzyme-intl": "^3.0.0",
-    "terra-toolkit": "^4.16.1",
-    "webpack-merge": "^4.1.2"
+    "terra-toolkit": "git+https://github.com/cerner/terra-toolkit.git#serve-updates",
```

Comment:
Should we update the tera-dev-site dependency as well to verify the correct version is being used?

Reply:
If the correct dependency wasn't getting used, all the wdio tests wouldn't have failed :)